zdm/MCMC_wrap2.py

```diff
@@ -0,0 +1,105 @@
+"""
```

Comment:
How is  this different from `MCMC_wrap.py`?
  do we need both?  

seems like a lot of duplicated code